src/main/scala/beam/agentsim/infrastructure/ChargingNetwork.scala

```diff
@@ -235,23 +259,35 @@ object ChargingNetwork {
   }
 
   final case class ChargingStation(zone: ParkingZone[_]) {
-    import ConnectionStatus._
-    private val connectedVehiclesInternal = mutable.HashMap.empty[Id[BeamVehicle], ChargingVehicle]
+    import ChargingStatus._
+    private val chargingVehiclesInternal = mutable.HashMap.empty[Id[BeamVehicle], ChargingVehicle]
 
-    private val waitingLineInternal: mutable.PriorityQueue[ChargingVehicle] =
+    private val gracedVehiclesInternal = mutable.HashMap.empty[Id[BeamVehicle], ChargingVehicle]
```

Comment:
Well. it is called Grace period. When a vehicle is done charging but still plugged to the wall. In this case some charging network companies might charge you for that. Actually the grace period, is the period that allowed to stay plugged without being charged for a penalty. After the garce period ends, they start charging you. Would you suggest to change the name?

Reply:
I will think on it, but I don't think there is anything better - becoming more aware of the domain is the solution for this really. So that is fine. Again, I'll think if there is a way to embed this directly into the name, but most of it is awareness.